apps/webapp/utils/validation.ts

```diff
@@ -0,0 +1,20 @@
+const BECH32_PREFIX = 'penumbrav2t';
+const ADDRESS_LENGTH = 146;
+
+export const validateAmount = (amount: string, balance: number): boolean =>
+  Boolean(amount) && Number(amount) > balance;
+
+//impl std::str::FromStr for Address {
+//   type Err = anyhow::Error;
+
+//   fn from_str(s: &str) -> Result<Self, Self::Err> {
+//       pb::Address {
+//           inner: bech32str::decode(s, bech32str::address::BECH32_PREFIX, bech32str::Bech32m)?,
+//           alt_bech32m: String::new(),
+//       }
+//       .try_into()
+//   }
+// }
```

Comment:
Instead of pasting code, just point to the lines in github: https://github.com/penumbra-zone/penumbra/blob/8d1644620779ddfd961e58f0f4703318b3d08910/crates/core/keys/src/address.rs#L201-L211